backend/drm/atomic.c

```diff
@@ -7,6 +7,9 @@
 #include "backend/drm/iface.h"
 #include "backend/drm/util.h"
 
+// connection property "Broadcast RGB" = "Full"
+#define DRM_MODE_BROADCAST_RGB_FULL 1
```

Comment:
The kernel patch uses `DRM_MODE_COLOR_RANGE_FULL` instead. Maybe rename to that and throw in an `#ifndef` in case it's made public.